conformance/conformance.proto

```diff
@@ -72,6 +72,10 @@ enum TestCategory {
                   // Opensource testees just skip it.
 }
 
+message FailureSet {
```

Comment:
I think you need to add the new message into https://github.com/protocolbuffers/protobuf/blob/master/conformance/conformance_php.php to fix the php's conformance test.

Reply:
I just added the new message, and it can pass php conformance test but it still failed in php_c for segmentation fault.  (See the commit for "fix php conformance test" test result)

Do you have any idea for where it may get segmentation fault?

Reply:
I mean you need to add require the newly added message here: https://github.com/protocolbuffers/protobuf/blob/master/conformance/conformance_php.php#L5

Reply:
Oh, sorry that I forgot this... Done